### DISCUSSION TITLE
Can too many background images decrease model performance?

### DISCUSSION THREAD

**Discussion by haimat** [ultralytics/yolov5#9211] [General]
I have a dataset with many background images (those without labels), at least 50% of all images in the dataset. Now I read in the YOLOv5 tutorials that you recommend about 10% of the whole dataset to be such background images. Now in my dataset it would be quite difficult to identify all those background images.

Thus, if a dataset includes that many background images, would that just extend training time, or would it also have a negative impact on the overall model training performance?

**Comment by glenn-jocher** [comment]
@haimat I don't think there's any ill effect of 'too many' background images. It depends on the priorities. To prioritise higher precision, use more background images, and to prioritize higher recall, use less.

**Reply by haimat** [reply]
Thanks!
